### ISSUE TITLE
2022 Rulebook Committee presentation example - require "chin jumping"

### ISSUE THREAD

**Issue by josiah-leinbach** [gryphonshafer/Quizzing-Rule-Book#155]
The rulebook contains only three requirements for jumping: 

- "Quizzers will be seated facing the audience and quizmaster" (3.5) 
- "The feet of the quizzers are to be on the floor while jumping" (3.5)
- "The use of the hands to assist in jumping; hands may not touch any part of the chair or the floor from the time "Question" is called until the quizmaster recognizes the 1st quizzer to jump" (4.5)

No where, however, do these stipulations state that the quizzer must be seated _on the bench_, something most quizzers do just out of convention and common sense. In fact, as currently written, "chin jumping" — a jumping style whereby the quizzer sits _behind_ and _underneath_ the bench with their hands in their lap and their chin holding down the seat — is completely legal.

I will even take this a step further: not only is "chin jumping" currently _permissible_ under the rules...it is _**preferable**_. 

Chin jumping offers a variety of advantages over normal jumping:

1. It eliminates the need for hand-buzzer or hand-assisted quizzing. All quizzers have a head, and thus all quizzers have a chin, so everyone can quiz by the same rules. Point for chin jumping.
2. No one likes it when quizzers rapidly click their seats up and down before a question. But if a quizzer tried to do that in the chin jumping stance, they would either get a migraine or throw up, and no quizzer wants that during a quiz. Again, point for chin jumping.
3. Finally, some quizzers, due to some combination of their height, weight, body frame, and muscular development, have a physiological advantage in jump speed over other quizzers. The variation in advantage with chin jumping is negligible if not nonexistent. Yet again, another point for chin jumping.

In short, I move that we amend the rulebook to _require_ the "chin jumping" style.

**Comment by jswingle** [comment]
I disagree with the idea of mandating the mandible, and all three of your advantages have flaws.

1. All quizzers must have a brain in order to quiz, but scientifically there is no guarantee in the future that the brain will need to be housed within a head. Therefore, not all quizzers have heads, so this doesn't solve the problem of needing to provide alternative jumping methods.

2. I like it when quizzers rapidly click their seats up and down before a question. So you are wrong when you say "no one likes it."

3. I think quizzers with more pronounced chins would have an unfair advantage, as they could put only the tip of their chin on the seat, while quizzers with smaller chins would have to put more of their face on the seat in order to jump with their chin.
4.

**Comment by shamwow2319** [comment]
Jeremy, I think you're letting your personal biases blind you to the brilliance of this idea. In the words of Taylor Swift, "Haters gonna hate hate hate hate hate"

**Comment by gryphonshafer** [comment]
Scientific study required to understand this better.

**Comment by levikoral** [comment]
I think that people should be able to do this if they want (taking the risk of possible injury which would be their fault) and if people think it to be unsanitary then they do not have to chin-jump.

**Comment by jswingle** [comment]
As fun as this was, I'm gonna close this out since we've already demo'd chin jumping and had our fun =)
